src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/CreateNewListModal.tsx

```diff
@@ -90,9 +119,24 @@ export const CreateNewListModal: React.FC<CreateNewListModalProps> = ({
       return null
     }
 
+    // const store = relayEnvironment.getStore()
+
+    // console.log('store', store.__records)
+
+    // const connectionID = ConnectionHandler.getConnectionID(
+    //   "-hardcoded-Key-of-the me",
+    //   "CollectorProfileSaves2Route_otherSaves"
+    // )
+
+    // console.log("[connectionID]", connectionID)
+
     try {
       const { createCollection } = await submitMutation({
-        variables: { input: { name: values.name } },
+        variables: {
+          input: { name: values.name },
+          // @ts-expect-error
+          connections: [connectionID],
```

Comment:
the problem is that we are not exactly able to get the `"-hardcoded-Key-of-the me",` here because we don't have direct access to the `store` we have access to the `relayEnvironment.getStore()` but it is of type `Store` while the `store` coming from updaters i.e. has type of `RecordSourceSelectorProxy`

Reply:
when I hardcoded the `"-hardcoded-Key-of-the me"` it works (but returns also an error) found the value of it with the help of relay dev tools
![Screenshot 2023-03-13 at 18 30 01](https://user-images.githubusercontent.com/21178754/224781263-19b263f3-1546-496a-8211-0e8e5d1b9c42.png)